Joinpoint Regression in R.R

```diff
@@ -199,149 +236,177 @@ pi.calc <- jp.calc(mdw$pi.n, mdw$pop, mdw$year)
 
 
 
 ### MAKING SENSE OF THE MODEL FITS ----
 
-# Remember that each model itself prints its output automatically, so by running the function that we wrote
-    # with each of the models inside of it, every time you run the function you should get the four model 
-    # outputs individually, along with the slope estimates for each of the segments of the fitted model and, 
-    # finally, the indexable tau (joinpoint) values. This is what we will use to visualize our data. 
+# Remember that each model itself prints its output automatically,
+# so by running the function that we wrote
+# with each of the models inside of it,
+# every time you run the function you should get the four model
+# outputs individually, along with the slope estimates for each of the
+# segments of the fitted model and, finally, the indexable tau (joinpoint)
+# values. This is what we will use to visualize our data.
 
+# when I simply print the name of the object,
+# what comes up is the list of joinpoint estimates.
 heart.calc
-    # when I simply print the name of the object, what comes up is the list of joinpoint estimates.
-    # you can index these easily:
-              heart.calc[[1]]
-              heart.calc[[2]]
-              heart.calc[[3]]
+# you can index these easily:
+heart.calc[[1]]
+heart.calc[[2]]
+heart.calc[[3]]
 
 
-# Let's make tables of all the joinpoint estimates               
+# Let's make tables of all the joinpoint estimates
 death.cause <- c("accidents", "cancer", "heart disease", "P&I", "stroke", "TB")
-one.joinpoint <- data.frame(accidents.calc[[1]], cancer.calc[[1]], heart.calc[[1]], pi.calc[[1]], stroke.calc[[1]], tb.calc[[1]])
+one.joinpoint <- data.frame(
+  accidents.calc[[1]], cancer.calc[[1]], heart.calc[[1]], pi.calc[[1]],
+  stroke.calc[[1]], tb.calc[[1]])
 colnames(one.joinpoint) <- death.cause
-two.joinpoints <- data.frame(accidents.calc[[2]], cancer.calc[[2]], heart.calc[[2]], pi.calc[[2]], stroke.calc[[2]], tb.calc[[2]])
+two.joinpoints <- data.frame(
+  accidents.calc[[2]], cancer.calc[[2]], heart.calc[[2]], pi.calc[[2]],
+  stroke.calc[[2]], tb.calc[[2]])
 colnames(two.joinpoints) <- death.cause
-three.joinpoints <- data.frame(accidents.calc[[3]], cancer.calc[[3]], heart.calc[[3]], pi.calc[[3]], stroke.calc[[3]], tb.calc[[3]])
+three.joinpoints <- data.frame(
+  accidents.calc[[3]], cancer.calc[[3]], heart.calc[[3]], pi.calc[[3]],
+  stroke.calc[[3]], tb.calc[[3]])
 colnames(three.joinpoints) <- death.cause
 
 jp.summary <- list(one.joinpoint, two.joinpoints, three.joinpoints)
 jp.summary
 
 # Time to visualize what we have here
-# Before we do that, it's important to remember that although we INPUT the data as number of deaths and 
-    # population size, the model used that information and fit them as though they were rates. So, we'll
-    # need to PLOT the log transformed mortality rates, and then visualize the joinpoint estimates
+# Before we do that, it's important to remember that although we INPUT the data
```

Comment:
Most of the plotting code is redundant so might want to consider placing within a function.